helix-control-m/2-external-monitoring-tools-examples/alerts-to-helix-itsm/README.md

```diff
@@ -65,7 +66,7 @@ Need to install the current fork of the remedy_py package. A pull request is pen
 * Helix Control-M
 * Automation API CLI
 
-> NOTE: It is likely compatible with Control-M on-premise systems, with the proper fields file (as per the documentation on the [Alerts template reference](https://docs.bmc.com/docs/display/ctmSaaSAPI/Alerts+Template+reference).
+> NOTE: It likely is compatible with Control-M on-premise systems, with the proper fields file (as per the documentation on the [Alerts template reference](https://docs.bmc.com/docs/display/ctmSaaSAPI/Alerts+Template+reference).
```

Comment:
I would avoid "likely compatible". If we haven't tested you can say that we haven't tested but it should work.

Reply:
Noted.